client/src/reducers/actions/index.js

```diff
@@ -1,18 +1,29 @@
-export const actionTypes =  {
-    CREATE_SESSION:"CREATE_SESSION"
-}
+export const actionTypes = {
+  CREATE_SESSION: "CREATE_SESSION",
+  DELETE_SESSION: "DELETE_SESSION",
+};
 /**
  * @function createSession
- * @param {string} message 
- * @param {string} token 
- * @param {string} firstname 
+ * @param {string} message
+ * @param {string} token
+ * @param {string} firstname
  * @param {string} lastname
+ * @param {int} id
  * @returns {object}  - Action object tipe CREATE_SESSION
  */
-export const createSession = (message,token,firstname,lastname) => ({
-    type : actionTypes.CREATE_SESSION,
-    message:message,
-    token : token,
-    firstname:firstname,
-    lastname:lastname
-});
\ No newline at end of file
+export const createSession = (message, token, firstname, lastname, id) => ({
+  type: actionTypes.CREATE_SESSION,
+  message: message,
+  token: token,
+  firstname: firstname,
+  lastname: lastname,
+  id: id,
+});
+export const deleteSession = (message, token, firstname, lastname, id) => ({
```

Comment:
Do you really need to know the message/token/firstname etc? It would make sense to remove all the parameters here and  set everything to an empty string